### PR TITLE
chore: add and run tests for check-configuration-keys

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Add tests for check-configuration-keys to ensure that some features won't change over time 

### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,6 @@ values.local.yaml
 
 # Website cache for the dependency checker
 cache
+
+# Python cache
+__pycache__
```
Comment (sumo-drosiek) [inline .gitignore:36]:
```suggestion
__pycache__

```

File: .github/workflows/pull_requests.yml [hunk 2]
```diff
@@ -53,6 +53,16 @@ jobs:
...
+  pr-name:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dylanvann/check-pull-request-title@v1
+        with:
+          # Match pull request titles in the form of conventional commit.
+          # rel: https://gist.github.com/marcojahn/482410b728c31b221b70ea6d2c433f0c
+          pattern: '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\([\w\-\.]+\))?(!)?: ([\w ])+([\s\S]*)'
+
```
Comment (swiatekm-sumo) [inline .github/workflows/pull_requests.yml:65]: This is from a different PR, right?

Comment (sumo-drosiek) [reply .github/workflows/pull_requests.yml:65]: yes, I thought I removed it :eye:

Comment (sumo-drosiek) [reply .github/workflows/pull_requests.yml]: forgoet to push

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**